.github/workflows/verify_consumer_pacts.yml

```diff
@@ -321,7 +321,7 @@ jobs:
                           -e JANITOR_CLIENT_CREDENTIAL_FILE_PATH="" \
                           -e JANITOR_TRACK_RESOURCE_PROJECT_ID="" \
                           -e JANITOR_TRACK_RESOURCE_TOPIC_ID="" \
-                          sbtscala/scala-sbt:eclipse-temurin-17.0.15_6_1.11.3_2.13.16 \
+                          sbtscala/scala-sbt:eclipse-temurin-17.0.15_6_1.11.5_2.13.16 \
```

Comment:
Dear Scala Steward,

This is the wrong way to update `sbt`. You missed a bunch of other docker images and you haven't actually updated sbt itself.

Sincerely,
me